From 869cfee70436d6f4b8e31d110ffb7957da210096 Mon Sep 17 00:00:00 2001
From: Jack Olivieri <jolivieri@anaconda.com>
Date: Sun, 23 Jun 2024 13:02:20 +0200
Subject: [PATCH] support ffmpeg

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ddc6dc15..a057afb6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,13 +52,13 @@ endif()


 # Options
-option(BUILD_SOX "Build libsox statically" ON)
+option(BUILD_SOX "Build libsox statically" OFF)
 option(BUILD_RIR "Enable RIR simulation" ON)
 option(BUILD_RNNT "Enable RNN transducer" ON)
 option(BUILD_ALIGN "Enable forced alignment" ON)
 option(BUILD_CUDA_CTC_DECODER "Build CUCTC decoder" OFF)
 option(BUILD_TORCHAUDIO_PYTHON_EXTENSION "Build Python extension" OFF)
-option(USE_FFMPEG "Enable ffmpeg-based features" OFF)
+option(USE_FFMPEG "Enable ffmpeg-based features" ON)
 option(USE_CUDA "Enable CUDA support" OFF)
 option(USE_ROCM "Enable ROCM support" OFF)
 option(USE_OPENMP "Enable OpenMP support" OFF)
--
2.40.1